src/afrolabs/components/kafka.clj

```diff
@@ -1401,14 +1470,20 @@
                                                 :d (s/and double?
                                                           #(< 0.0 % 1.0))))
 (s/def ::recreate-topics-with-bad-config boolean?)
+(s/def ::update-topics-with-bad-config boolean?)
+(s/def ::ktable-compaction-policy #{"compact"
+                                    "delete,compact"
+                                    "compact,delete"})
```

Comment:
The `cleanup.policy` of `compact,delete` is a ktable that will expire records that don't receive updates. This is useful for _state_, but not for something like fleet or vehicle config.

Reply:
This famous documentation, i like it